unblob/handlers/compression/gzip.py

```diff
@@ -124,3 +147,29 @@ def calculate_chunk(self, file: File, start_offset: int) -> Optional[ValidChunk]
             start_offset=start_offset,
             end_offset=file.tell(),
         )
+
+
+class MultiVolumeGzipHandler(DirectoryHandler):
+    NAME = "multi-gzip"
+    EXTRACTOR = MultiGZIPExtractor()
+
+    PATTERN = Glob("*.gz.aa")
```

Comment:
I was wondering if there is a reason restricting support only to `split`'s default numbering, which excludes both numeric and hex numbering (`*.gz.00`), as well as explicitly set, non-default suffix length (`*.gz.000`) , and additional suffixes (`*.gz.aa-suffix`) .

Could this pattern be generalized to
```suggestion
    PATTERN = Glob("*.gz.*")
```
?

Reply:
Since the `paths` list would be sorted, we can definitely use that pattern and not restrict on a specific numbering.